js/repeater-list.js

```diff
@@ -452,6 +492,19 @@
 		}
 		$row.append($col);
 
+		if (this.viewOptions.list_selectable === 'multi' && columns[columnIndex].property === '@_CHECKBOX_@') {
+			var checkBoxMarkup = '<label class="checkbox-custom checkbox-inline body-checkbox" data-initialize="checkbox">' +
+				'<input class="sr-only" type="checkbox"></label>';
```

Comment:
Although I'm not aware of any side affects, we may not want to not promote the use of `data-initialize="checkbox"` and manually use `$().checkbox();`. I do know that we ran into issues with auto initializing in scheduler. This will also allow us to determine when the checkbox initializes instead of waiting for a mouseover.